### ISSUE TITLE
Handle error conditions in controller

### ISSUE THREAD

**Issue by gregdeane** [zalando-incubator/solution-center-feedback#30]
There are two `.catch` blocks that we are not truly handling. I've spoken to Sherif and asked that he returns an error code and message. This will allow us to display errors.

In some cases (401, 500) we won't have a nicely formatted error and should default to a generic message.

**Comment by dami-gg** [comment]
@gabrielhl should be involved in this and tell us where and how to display the errors to the user

**Comment by gregdeane** [comment]
@gabrielhl We will be dealing with two types of errors: errors where we know specifically what happened (e.g. 401 User does not have access to this module) and unknown errors (e.g. 500 Internal Service Error).

Please consider both types of errors in your suggestions.

**Comment by dami-gg** [comment]
Waiting also for this to be done by backend:

https://github.bus.zalan.do/norris/module-service/issues/25

**Comment by gregdeane** [comment]
Per @gabrielhl 

For any error that can be caught:

1. Display an error message: "It seems we are not able to record your feedback right now. We'll ask again later. Thanks!"
1. Hide the tool for this session.

**Comment by gregdeane** [comment]
@gabrielhl the length of the message above is too long for the amount of time that the error message is shown (4 seconds). There is not enough time to read it for the average person.

I spoke to @gloriarupprecht and we came up with a shortened version. This is easier to read in the 4s time allotment.

Since the message will rarely be shown, we felt that this was appropriate. I have made the update in the code.

**Updated Message**
"We were unable to record your feedback. We'll ask again later."
